src/client/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -167,10 +167,8 @@ private void UpdateAppMetadata(string clientId, string environment, string famil
             if (!requestParams.IsClientCredentialRequest)
```

Comment:
Can probably combine this further, if not for the log on line 165

Reply:
Perhaps the log at line 160 is sufficient?